chase/frontend/components/whiteboard.tsx

```diff
@@ -0,0 +1,68 @@
+import { Editor } from "primereact/editor";
+import React from "react";
+
+/**
+ * This Component is a wrapper for the PrimeReact Editor Component.
+ * It allows easy usage of the Editor Component as a Whiteboard.
+ * @param value The value (content / text) of the Whiteboard in html format
+ * @param readOnly Whether the Whiteboard is read only or not. When readOnly is true, the Header of the Editor is hidden.
+ * @param setContentFunction The function that is called when the content of the Whiteboard changes
+ * @param rest The rest of the props that are passed to the PrimeReact Editor Component
+ * @returns A Whiteboard Component
+ */
+
+export default function ChairWhiteboard({
+  value = "Hello World!",
+  readOnly = false,
+  setContentFunction,
+  ...rest
+}: {
+  value?: string;
+  readOnly?: boolean;
+  setContentFunction?: (content: string) => void;
+  [key: string]: any;
```

Comment:
How can I avoid this any key so that the linter does not complain? I have no Idea how to pass through other params to the Prime React Component while remaining typesafe other than using any in this context...

Reply:
The unknown type could be what you're searching for: https://stackoverflow.com/questions/51439843/unknown-vs-any